src/components/redirect_notification/SupportedNotification_NotInstalled.js

```diff
@@ -22,26 +24,39 @@ export default function SupportedNotification_NotInstalled({
     <Modal open={open} onClose={handleCancel}>
       <Header>
         <Heading>
-          <Icon src={"../static/images/zeeguuLogo.svg"} />
-          The Zeeguu Reader<br></br>browser extension is not installed
+          Enable translating foreign articles with&nbsp;the&nbsp;Zeeguu
+          browser&nbsp;
+          {runningInChromeDesktop() ? "extension" : "add-on"}
         </Heading>
       </Header>
       <Main>
-        <p>
-          For the best user experience we recommend you to read articles with{" "}
-          <s.Strong>The Zeeguu Reader</s.Strong> browser extension.
-        </p>
-        <p>
-          To read this article with the help of Zeeguu without the extension,
-          simply click "Add to Saves" above the article's title.
+        <FullWidthImage
```

Comment:
I can see that with the Header you have dropped it as a component an instead you import the styling which you then pass the props as normal. I can see that the `FullWidthImage` is essentially the same. Wouldn't it be better to just import the styling and not be dependent on the having the props setup in the component? 

I think this was similar to what happened with the `Button` Component which didn't have a Prop for type and then it didn't work when I was trying to assign the `type="submit"`, and if it was just a styled component we wouldn't have this problem. I see you have also deleted the `Button`. So what's the pattern here?